yjit.c

```diff
@@ -289,6 +294,24 @@ rb_vm_ci_flag(const struct rb_callinfo *ci) {
     return vm_ci_flag(ci);
 }
 
+const struct rb_callinfo_kwarg *
+rb_vm_ci_kwarg(const struct rb_callinfo *ci)
+{
+    return vm_ci_kwarg(ci);
+}
+
+int
+rb_get_cikw_keyword_len(const struct rb_callinfo_kwarg* cikw)
+{
+    return cikw->keyword_len;
+}
+
+VALUE
+rb_get_cikw_keywords_idx(const struct rb_callinfo_kwarg* cikw, int idx)
+{
+    return cikw->keywords[idx];
+}
```

Comment:
rb_callinfo gets exported in this PR, yeah. But unfortunately, rb_callinfo_kwarg has to be opaque because it has an unspecified-size VALUE array as its last member, which causes bindgen to emit a lot of nontrivial code that I assume we don't want.